### ISSUE TITLE
Not able to install nlm_ingestor

### ISSUE THREAD

**Issue by sli701** [nlmatics/nlm-ingestor#30]
![image](https://github.com/nlmatics/nlm-ingestor/assets/22850666/34dc9113-284c-4a51-9ba0-b89819897857)

Hi, I have been trying to install nlm-ingestor, it keeps saying the above errors, any hints would be greatly appreciated

**Comment by siliconberry** [comment]
yeah, same here.. spent 5+ hrs on it and still no luck.. Frustrating !

**Comment by ansukla** [comment]
libxml needs some dependencies to be installed outside pip. See: https://lxml.de/installation.html#requirements. This can get  alittle more complex on Macs or windows. Using the docker image is easier.

**Comment by capoveur** [comment]
Hi any update on this? How does this work using the docker image?
